styles/globals.css

```diff
@@ -1,8 +1,10 @@
-html,
 body {
-  padding: 0;
+  padding-left: 8%;
+  padding-right: 8%;
+  padding-top: 70px;
   margin: 0;
-  font-family: 'Montserrat', sans-serif;
+  font-family: 'Montserrat', sans-serif !important;
```

Comment:
what was the use of !important here?

Reply:
The Montserrat font wasn't actually showing until I put it in

Reply:
ahh ok, we can investigate this later